src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/CalculateContamination.java

```diff
@@ -32,19 +32,34 @@
  *     The resulting contamination table is used with {@link FilterMutectCalls}.
  * </p>
  *
- * <p>This tool and GetPileupSummaries together replace GATK3's ContEst.</p>
+ * <p>This tool and GetPileupSummaries together replace GATK3's ContEst.  Like ContEst, this tool estimates contamination based on the signal
+ * from ref reads at hom alt sites.  However, ContEst uses a probabilistic model that assumes a diploid genotype with no copy number
+ * variation and independent contaminating reads.  That is, ContEst assumes that each contaminating read is drawn randomly and
+ * independently from a different human.  This tool uses a simpler estimate of contamination that relaxes these assumptions.  In particular,
+ * it works in the presence of copy number variations and with an arbitrary number of contaminating samples.  In addition, this tool
+ * is designed to work well with no matched normal data.  However, one can run {@link GetPileupSummaries} on a matched normal bam file
```

Comment:
This + matched normal is new! I'm excited to try it out.

Reply:
Yes it is.